### ISSUE TITLE
Get a list of all published BXD phenotypes

### ISSUE THREAD

**Issue by DannyArends** [rqtl/GNapi#11]
Hey Karl,

I used the GNapi to get a list of all published phenotypes before, but it seems that the API cannot do this anymore.

I can still get individual phenotypes using:
list_datasets("BXD", "10001")

However it doesn't seem possible to get a list of all phenotype datasets, since 
list_datasets("BXD")

doesn't show the them anymore.

Is there a way to get all phenotype IDs "10001", "10002", etc, so I can query them ?

**Comment by kbroman** [comment]
Yes, I think it was a change in the [GeneNetwork API](https://github.com/genenetwork/genenetwork2/blob/testing/api_readme.md).

**Comment by DannyArends** [comment]
Hmm, I don't seems to be able to query for all phenotype IDs, once I have an ID I can query the information about it. Seems this was changed at some point.

I'll email @pjotrp / Zach to see if it's possible to get a list of all phenotype IDs and short descriptions to query them
